ql/src/test/results/clientpositive/llap/masking_acid_no_masking.q.out

```diff
@@ -54,8 +53,9 @@ POSTHOOK: Input: default@masking_acid_no_masking
 POSTHOOK: Input: default@nonacid_n0
 POSTHOOK: Output: default@masking_acid_no_masking
 POSTHOOK: Output: default@masking_acid_no_masking
-POSTHOOK: Output: default@masking_acid_no_masking
 POSTHOOK: Output: default@merge_tmp_table
 POSTHOOK: Lineage: masking_acid_no_masking.key SIMPLE [(nonacid_n0)s.FieldSchema(name:key, type:int, comment:null), ]
+POSTHOOK: Lineage: masking_acid_no_masking.key SIMPLE [(nonacid_n0)s.FieldSchema(name:key, type:int, comment:null), ]
```

Comment:
is this a duplicate?

Reply:
These lineages are generated by the MoveTask when inserting.

By turning on `hive.merge.split.update` update branch of merge statements are splitted into a insert and a delete branch.

Originally this merge had only one insert branch but now it has two to the same table same columns:
- one for the insert branch
- one for the update branch